submarine-server/server-core/src/test/java/org/apache/submarine/server/metastore/rest/MetaStoreApiTest.java

```diff
@@ -0,0 +1,180 @@
+    Map<String, String> map = new HashMap<>();
+    map.put("key", "value");
+    database.setParameters(map);
+    database.setOwnerName("root");
+    database.setOwnerType(PrincipalType.USER);
+
+    Gson gson = new Gson();
+    String databaseJson = gson.toJson(database);
+
+    metaStoreApi.createDatabase(databaseJson);
+    Response databaseCountResponse = metaStoreApi.getDatabaseCount();
+    assertEquals(databaseCountResponse.getStatus(), Response.Status.OK.getStatusCode());
+    assertTrue(((String) databaseCountResponse.getEntity()).contains("\"result\":1"));
+
+    Table table = new Table();
+    table.setTableName("testtable");
+    table.setDbName("testdb");
+    table.setOwner("root");
+    table.setCreateTime((int) new java.util.Date().getTime() / 1000);
+    table.setLastAccessTime((int) new Date().getTime() / 1000);
+    table.setRetention(0);
+    StorageDescriptor sd = new StorageDescriptor();
+    List<FieldSchema> fieldSchemas = new ArrayList<>();
+    FieldSchema fieldSchema = new FieldSchema();
+    fieldSchema.setName("a");
+    fieldSchema.setType("int");
+    fieldSchema.setComment("a");
+    fieldSchemas.add(fieldSchema);
+    sd.setCols(fieldSchemas);
+    sd.setLocation("hdfs://mycluster/user/hive/warehouse/testdb.db/testtable");
+    sd.setInputFormat("org.apache.hadoop.mapred.TextInputFormat");
+    sd.setOutputFormat("org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat");
+    sd.setCompressed(false);
+    sd.setNumBuckets(-1);
+    SerDeInfo serdeInfo = new SerDeInfo();
+    serdeInfo.setName("test");
+    serdeInfo.setSerializationLib("org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe");
+    Map<String, String> parametersMap = new HashMap<>();
+    parametersMap.put("serialization.format", "|");
+    parametersMap.put("field.delim", "|");
+    serdeInfo.setParameters(parametersMap);
+    sd.setSerdeInfo(serdeInfo);
+    table.setSd(sd);
+    List<FieldSchema> partitionKeys = new ArrayList<>();
+    table.setPartitionKeys(partitionKeys);
+    Map<String, String> parameters = new HashMap<>();
+    table.setParameters(parameters);
+    String viewOriginalText = "";
+    table.setViewOriginalText(viewOriginalText);
+    String viewExpandedText = "";
+    table.setViewExpandedText(viewExpandedText);
+    String tableType = "MANAGED_TABLE";
+    table.setTableType(tableType);
+
+    String tableJson = gson.toJson(table);
+    metaStoreApi.createTable(tableJson);
+
+    Response tableResponse = metaStoreApi.getTable("testdb", "testtable");
+    assertEquals(tableResponse.getStatus(), Response.Status.OK.getStatusCode());
+    assertTrue(((String) tableResponse.getEntity()).contains("\"tableName\":\"testtable\""));
+    Response tableCountResponse = metaStoreApi.getTableCount();
+    assertEquals(tableCountResponse.getStatus(), Response.Status.OK.getStatusCode());
+    assertTrue(((String) tableCountResponse.getEntity()).contains("\"result\":1"));
+  }
+
+  @After
```

Comment:
Maybe need change to `@AfterClass`?
Because it only needs to be remve all record once.